src/OSPSuite.UI/Controls/UxHtmlLabel.cs

```diff
@@ -1,51 +1,40 @@
 using System;
 using System.Drawing;
+using System.Windows.Forms;
 using DevExpress.Utils.Controls;
 using DevExpress.XtraEditors.Controls;
 using DevExpress.XtraRichEdit;
+using AutoSizeMode = DevExpress.XtraRichEdit.AutoSizeMode;
 
 namespace OSPSuite.UI.Controls
 {
    public class UxHtmlLabel : UxRichEditControl, IXtraResizableControl
    {
-      private static readonly object _layoutInfoChanged = new object();
-      private static string _fontFamily;
-      private static double _fontSize;
+      private readonly object _layoutInfoChanged = new object();
+      public string FontFamily { get; set; } = UIConstants.DEFAULT_HTML_FONT;
```

Comment:
Those properties should be available as set so that they can be updated from other components